.sonar/report-task.txt

```diff
@@ -0,0 +1,5 @@
+projectKey=bip4cast
```

Comment:
`.sonar` directory should be ignored from versioning, for same reasons we removed .idea and binaries.